planq/chain.json

```diff
@@ -65,6 +70,20 @@
           "darwin/arm64": "https://github.com/planq-network/planq/releases/download/v1.0.3/planq_1.0.3_Darwin_arm64.tar.gz",
           "windows/amd64": "https://github.com/planq-network/planq/releases/download/v1.0.3/planq_1.0.3_Windows_x86_64.zip"
         }
+      },
+      {
+        "name": "v1.0.7",
+        "recommended_version": "v1.0.7",
+        "compatible_versions": [
+          "v1.0.7"
+        ],
```

Comment:
can we add the cosmos_sdk_version, consensus, and ibc_go_version into the v1.0.7 version object please?

Reply:
added https://github.com/cosmos/chain-registry/pull/2663/commits/2acdf9fa1aacc451e8d541b4ca1f40f1ce8a7f13